machine-learning/README.md

```diff
@@ -1,4 +1,99 @@
-# Machine Learning (PyTorch, Tensorflow)
-all the tools and guidance needed to get up and running at a hackathon!
+# Machine Learning (Keras, PyTorch, Tensorflow)
+
+## What is Machine Learning and Why Would I Use it?
+Machine Learning is a broad umbrella for methods of developing artifial intelligence, which is generally used to find patterns in large data sets based on mathematical and statistical calculations. This becomes bery useful when a phenomena can not be modelled by a deterministic algorithm.   
+
+In the context of a Hackathon, Machine Learning is designed to target a single problem with specific input parameters than can be consistently replicated on a large scale and thus, is most commonly used to develop an market solution tailored to a specific company challenge or a specific social problem. 
+
+The great thing about Machine Learning, is that it is applicable to most modern day problems. This ranges from economics, where ML is being used to developing models to predict stock prices (quantative analysis, a very lucrative field), to the medical field, where ML is being used to analyze X-Rays to determine if a patient has cancer.
+
+Anything where normally we use our intution to predict a pattern, ML can be applied to do the work quicker and better!
+
+## Example Projects
+Here are a couple of the cool ML hackathon projects as some inspiration to get started!
+
+1. [NavAssistAI](https://devpost.com/software/navassistai) - This project won the Tensorflow World 2.0 Challenge, and attempts to help visually impaired individuals cross the street using machine learning. I like this project because its very applicable and usable in real life. 
+
+2. [Coughvid](https://devpost.com/software/coughvid-ai-based-app-for-covid-19-screening-using-coughing) - This project placed 3rd at LauzHack for COVID. It uses ML to analyze coughing sounds and symptoms to predict if a patient has COVID-19.
+
+3. [ColorMe](https://devpost.com/software/colorme-50fz68) - A unique project which usual Image Analysis to predict what color palettes for clothing would look best on the user.
+
+4. [StickyAR](https://devpost.com/software/stickyar) - Fun game built at HackMIT that takes a more light-hearted approach to ML. These hackers used OpenCV to detect sticky notes in a frame, and let the end user play a game using those sticky notes!
+
+There are a lot more! Browse [devpost](https://devpost.com) to see more cool projects. 
+
+## Tools and libraries to get you started
+If you're gonna do any kind of ML project, I'd highly recommend Python. Many common Python ML libraries such as Keras, PyTorch, and Tensorflow abstract alot of the process for you, so you can focus on the bigger picture! This is especially good for beginners who may not understand everything that is going on behind the scenes just yet. C++ and Java are also pretty popular options, but I can't speak to their usability.
```

Comment:
```suggestion
If you're gonna do any kind of ML project, I'd highly recommend Python. Many common Python ML libraries such as Keras, PyTorch, and Tensorflow abstract a lot of the process (math) for you so that you can focus on the bigger picture! This is especially good for beginners who may not understand everything that is going on behind the scenes just yet. C++ and Java are also pretty popular options, but I can't speak to their usability.
```